frontend/src/metabase/css/reset.module.css

```diff
@@ -37,15 +37,12 @@
     color: inherit;
     cursor: pointer;
     text-decoration: none;
-
-    &:focus {
-      outline: 2px solid var(--mb-color-focus);
-    }
   }
 
   button,
+  a,
   [role="button"] {
-    &:focus {
+    &:focus-visible {
```

Comment:
This change limits the focus outlines to keyboard navigation, rather than whenever something is clicked